### PR TITLE
Implement combined creep and instantaneous plasticity.

### PR DESCRIPTION
**Pull request by jasondhales**
Implement combined creep and instantaneous plasticity.  #6978

### CODE REVIEW

File: modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C [hunk 1]
```diff
@@ -6,47 +6,137 @@
...
+                                    stress_new);
+    }
+
+    // now check convergence
+    RankTwoTensor deltaS(stress_new_last - stress_new);
+    delS = std::sqrt(deltaS.doubleContraction(deltaS));
+    if (counter == 0)
+    {
+      first_delS = delS;
+    }
```
Comment (bwspenc) [inline modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C]: One liner with braces.

File: modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C [hunk 2]
```diff
@@ -6,47 +6,137 @@
...
+
+    ++counter;
+  }
+
+  if (counter == _max_its &&
+     delS > _absolute_tolerance &&
+     (delS/first_delS) > _relative_tolerance)
+  {
+    mooseError("Max stress iteration hit during ComputeReturnMappingStress solve!");
+  }
```
Comment (bwspenc) [inline modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C]: One liner with braces.

**Comment by bwspenc** [comment]
This code looks great.  I think it's ready to go once those minor issues are resolved.

**Comment by permcody** [comment]
@jasondhales - Were you aware this change breaks BISON? Do you have a patch to get us going again?

**Comment by friedmud** [comment]
My fault for merging it... I thought it was ready to go

**Comment by jasondhales** [comment]
I had a pull request for Bison that was merged sometime last night.  They must have collided.  Where do I find the build error?

**Comment by permcody** [comment]
It's currently the topmost "red" box on moosebuild.org. "App tests: Test Bison".

**Comment by jasondhales** [comment]
I have a merge request in for bison that will be accepted as soon as the build completes.  It just turns off those tests.  After the moose update is complete, I'll turn them back on with updated input files.

**Comment by permcody** [comment]
OK - I'll watch for the BISON master merge and will invalidate the failed
build.

**Pull request merged**